pkg/cmd/runtime/plugins/k8s/k8s_runtime.go

```diff
@@ -21,3 +37,167 @@ func (p *K8sRuntime) Name() string { return "k8s" }
 func (p *K8sRuntime) Run(app string, args ...string) error {
 	return errors.New("TODO")
 }
+
+func (p *K8sRuntime) GetDefaultClient() (clientset *kubernetes.Clientset, err error) {
```

Comment:
if the `GetDefaultClient` is the private method, it should be `getDefaultClient`, the first letter is lower case.

Reply:
ok